### PR TITLE
readme updated according to issue #1861

### LINKED ISSUE #1861: Fix typo
**Current Behavior**

Require some changes in Readme.md over Community and Contributing section.
**Details :**

tempsnip

**Desired Situation**

- 1st box : Need two different links to be open, by clicking on "community drive" & "Community Member Form" separately.
- 2nd box : Need to configure texts correctly.

##### (Follow the <a href="https://github.com/layer5io/meshery/blob/master/README.md">reference</a>, if required.)
---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by soham4abc**
**Documentation update**

This PR fixes #1861 

**Notes for Reviewers**
Pls check the readme updated as per requests.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,16 +101,16 @@ Jump into our <a href="http://slack.layer5.io"><img src=".github/assets/images/s
...
 ✔️ <em><strong>Join</strong></em> any or all of the weekly meetings on the <a href="https://calender.google.com/calendar/b/1?cid=bGF5ZXI1LmlvX2VoMmFhOWRwZjFnNDBlbHZvYzc2MmpucGhzQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20">community calendar</a>.<br />
 ✔️ <em><strong>Watch</strong></em> community <a href="https://www.youtube.com/playlist?list=PL3A-A6hPO2IMPPqVjuzgqNU5xwnFFn3n0">meeting recordings</a>.<br />
-✔️ <em><strong>To Access </strong></em><a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">community drive, fill Community Member Form</a>.<br />
+✔️ <em><strong>To Access community drive,</strong></em><a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA"> fill Community Member Form</a>.<br />
 <br />
 </p>
 <p>
 <i>Not sure where to start?</i> Grab an open issue with the <a href="https://github.com/issues?utf8=✓&q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22help+wanted%22+">help-wanted label</a>
 </p>
 <p>Find us on Twitter: <a href="https://twitter.com/layer5"><img src=".github/assets/images/twitter.png" height="16px"
```
Comment (Nikhil-Ladha) [inline README.md:110]: Put the image before ":". So that it's common for all of them.

File: README.md [hunk 2]
```diff
@@ -103,16 +101,16 @@ Jump into our <a href="http://slack.layer5.io"><img src=".github/assets/images/s
...
-align="bottom"/>@layer5</a>,<a href="https://twitter.com/mesheryio">@mesheryio</a>&<a href="https://twitter.com/smp_spec">@smp_spec</a>. Subscribe on <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1">Youtube</a>
+align="bottom"/>@layer5</a>, <a href="https://twitter.com/mesheryio">@mesheryio</a> and <a href="https://twitter.com/smp_spec">@smp_spec</a>. <br>Subscribe on <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1">Youtube</a>
```
Comment (Nikhil-Ladha) [inline README.md:111]: nit: Subscribe us on Youtube.

File: README.md (hunk 1 above)
Comment (soham4abc) [reply README.md:110]: could you pls explain a bit, Sorry I didn't get you.

File: README.md (hunk 2 above)
Comment (soham4abc) [reply README.md:111]: Ok on it

**Comment by l5io** [comment]
🚀 Preview for commit 643a73536dc52332712296e538d6e34b73920360 at: https://60a6532990bb9c2ecde2cc22--layer5ng.netlify.app

File: README.md (hunk 1 above)
Comment (Nikhil-Ladha) [reply README.md:110]: Make this
```
<p>Find us on Twitter: <a href="https://twitter.com/layer5"><img src=".github/assets/images/twitter.png" height="16px" align="bottom />
```
like this
```
<p>Find us on Twitter <img src=".github/assets/images/twitter.png" height="16px" align="bottom" />:```
And put the link only around the `@layer5` text.

Comment (soham4abc) [reply README.md:110]: thank you got it!

File: README.md (hunk 2 above)
Comment (Nikhil-Ladha) [reply README.md:111]: Actually, leave it. Let it be this way only. Only fix the above comment.

File: README.md (hunk 1 above)
Comment (leecalcote) [reply README.md:110]: Gents, Could we make line 104 match verbatim the same line in https://github.com/layer5io/layer5-repo-template‘s readme?

File: README.md (hunk 2 above)
Comment (soham4abc) [reply README.md:111]: ok

File: README.md (hunk 1 above)
Comment (Nikhil-Ladha) [reply README.md:110]: Yeah, that would be better.

Comment (soham4abc) [reply README.md]: ok on it

Comment (soham4abc) [reply README.md]: @leecalcote  I have done the requested changes pls see to it.

File: README.md [hunk 3]
```diff
@@ -103,16 +101,15 @@ Jump into our <a href="http://slack.layer5.io"><img src=".github/assets/images/s
...
+✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">Community Drive</a> by completing a community <a href="https://layer5.io/newcomer">Member Form</a>.<br />
 <br />
 </p>
 <p>
 <i>Not sure where to start?</i> Grab an open issue with the <a href="https://github.com/issues?utf8=✓&q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22help+wanted%22+">help-wanted label</a>
 </p>
-<p>Find us on Twitter: <a href="https://twitter.com/layer5"><img src=".github/assets/images/twitter.png" height="16px"
-align="bottom"/>@layer5</a>,<a href="https://twitter.com/mesheryio">@mesheryio</a>&<a href="https://twitter.com/smp_spec">@smp_spec</a>. Subscribe on <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1">Youtube</a>
+<p>Find us on Twitter <img src=".github/assets/images/twitter.png" height="16px" align="bottom" />: @layer5</a>, <a href="https://twitter.com/mesheryio">@mesheryio</a> and <a href="https://twitter.com/smp_spec">@smp_spec</a>. <br>Subscribe on <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1">Youtube</a>
```
[the remark this answers is not in the document]
Comment (soham4abc) [reply README.md]: resolved

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**